docs/UserGuide.md

```diff
@@ -33,7 +33,7 @@ SellSavvy is **optimized for users who prefer Command Line Interface** (CLI) whi
 
    * `addcustomer n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a customer named `John Doe` to the displayed customer list.
 
-   * `deletecustomer 3` : Deletes the 3rd customer contact shown in the displayed customer list.
+   * `deletecustomer 3` : Deletes the 3rd customer's contact and order list shown in the displayed customer list and order list.
```

Comment:
no we doesn't delete the displayed order list. we delete his list not the displayed.